pkg/vault/test/foundry/unit/VaultCommonBasicFunctions.t.sol

```diff
@@ -402,4 +402,22 @@ contract VaultCommonBasicFunctionsTest is BaseVaultTest {
         vm.expectRevert(abi.encodeWithSelector(PoolConfigLib.InvalidPercentage.selector, MAX_FEE_PERCENTAGE + 1));
         vault.manualSetStaticSwapFeePercentage(pool, MAX_FEE_PERCENTAGE + 1);
     }
+
+    function testFindTokenIndex__Fuzz(address[8] memory tokensRaw, uint256 tokenIndex, uint256 length) public view {
```

Comment:
Could add an explicit test that makes it revert, too. (We're triggering it indirectly in `testGetSingleInputArray`.)